package.json

```diff
@@ -20,8 +20,8 @@
     "src"
   ],
   "keywords": [],
-  "system": {
-    "main": "donejs-chat/index.stache!done-autorender",
+  "steal": {
+    "main": "donejs-chat/index.stache!done-autorender/no-zone",
```

Comment:
Why the addition of `no-zone`?

Reply:
@chasenlehara in a test it showed that steal-socket.io is not working as expected anymore this example fails without that.

i found many reasons why that could happen i simply did what is the most easy to get that running to use it for training people.

but your right this is worth creating a issue if we wan't that all works as it got designed.

my personal opinion about that is simply use socket.io and point out to never use that without no-zone.